index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel="stylesheet">
+</head>
+<body>
+    <main>
+        <header>
+            <a href="#">My logo</a>
+            <div>
+             <i class="fas fa-bars"></i>
+            </div>
+        </header>
+        <section>
+            <h1 class="profile">I'm Dare Glad to see you!</h1> 
+            <p class="dev">
+             I’m a software developer! I can help you build a product , feature or 
+             website Look through some of my work and experience! If you like what 
+             you see and have a project you need coded, don’t hestiate to contact 
+             me.
+            </p>
+            <div><a href="#" class="connect" >LET'S CONNECT</a></div>
+            <ul>
+                <li>
+                    <a href="#">
+                        <img src="./images/Twitter-logo.png" class="pix" alt="twitter">
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Twitter-logo.png" class="pix" alt="linkedin">
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                       <img src="./images/M-logo.png" class="pix" alt="letter m">
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                       <img src="./images/Github-logo.png" class="pix" alt="github">
+                    </a>
+               </li>
+               <li>
+                   <a href="#">
+                       <img src="./images/fingers-logo.png" class="pix" alt="finger icon">
+                   </a>
+                </li>
+            </ul>
+        </section>
+    </main>
+    
```

Comment:
- [ ] Please delete this blank line right here to keep your code completely clean.
Kindly remember that there should be no blank lines or indentations unless for a valid reason such as separating two pieces of logical code or two large sections of code.